uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/test/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemImplTest.java

```diff
@@ -92,6 +94,11 @@ public void testRemoteRoot() throws IOException, GitAPIException {
         assertThat(root.getRoot().toString()).isEqualTo("/");
     }
 
+    private JGitFileSystemLock createFSLock(Git git) {
+        return new JGitFileSystemLock(git,
+                                      30000l);
```

Comment:
Does this numeric literal end with `1` or `l`? It's recommended to use `L` at the end of `long` numeric literals to avoid this kind of ambiguity. Also since java 7 you can use [underscores in numeric literals](https://docs.oracle.com/javase/7/docs/technotes/guides/language/underscores-literals.html) for better readability. -> `30_000L`